docs/IE_PLUGIN_DG/NewTransformation.md

```diff
@@ -353,26 +392,34 @@ NGRAPH_ENABLE_VISUALIZE_TRACING=1 -  enables visualization after each transforma
 
 This topic mostly related to conversion to legacy opset and plugins that based on CNNNetwork but still this mechanism can be applied for other cases.
 Let's suppose that plugin X enabled `opset3::StridedSlice` operation support and you want to disable `ngraph::pass::ConvertStridedSliceToCrop` transformation for plugin X.
-To do this you need to extend transformation class with `ngraph::pass::PassParam` class. This class extends transformations class with `transformation_callback` that can be set by plugin that uses legacy conversion. 
+To do this you need to create callback on plugin side and pass it to transformation. And also you need to update particular transformation to use this callback.  
 
 ```cpp
-// Extend transformation class with PassParam
-class ngraph::pass::ConvertStridedSliceToCrop: public ngraph::pass::GraphRewrite, public ngraph::pass::PassParam {
-    ...
-}
-
-// Update callback to be able to use transformation_callback if this transformation based on GraphRewrite.
+// Update callback to be able to use m_transformation_callback if this transformation based on GraphRewrite.
 ngraph::graph_rewrite_callback callback = [this](pattern::Matcher &m) {
     ...
 }
 
-// Use transformation_callback not to execute transformation
-if (transformation_callback(node)) {
+// Use transformation_callback not to execute transformation if callback returns true fro given node
+if (m_transformation_callback(node)) {
     return false;
 }
-```
 
-TODO: link to existing example
+// Implement transformation callback and pass it directly to transformation or pass::Manager
+const auto transformations_callback = [](const std::shared_ptr<const ::ngraph::Node> &node) -> bool {
+    return std::dynamic_pointer_cast<const ::ngraph::opset3::StridedSlice>(node) != nullptr;
+};
+
+// Register transformations and pass callback to pass::Manager
+ngraph::pass::Manager manager;
+manager.register_pass<ngraph::pass::CommonOptimizations>();
+...
+// ConvertOpSet1ToLegacy contains ConvertStridedSliceToCrop transformation
+manager.register_pass<ngraph::pass::ConvertOpSet1ToLegacy>();
```

Comment:
Do we need to demonstrate usage of legacy passes in docs? I suppose - no

Reply:
removed.